src/game.cpp

```diff
@@ -7685,8 +7685,12 @@ void game::examine( const tripoint &examp )
                 return;
             }
         }
+        if( mon != nullptr && mon->has_flag( MF_MILKABLE ) ) {
+            add_msg( m_info, _( "This is a milkable cow." ) );
```

Comment:
But here it *explicitly* says "cow" and there is now way to change this. Even if I would add the flag to rabbits, it would say "This is a milkable cow."